client/src/components/BecomeInterpreterForm.js

```diff
@@ -0,0 +1,167 @@
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
+import { Button, Form, Dropdown } from "semantic-ui-react";
+import { Grid } from "semantic-ui-react";
+
+const languageOptions = [
+  { key: "Arabic", text: "Arabic", value: "Arabic" },
+  { key: "Chinese", text: "Chinese", value: "Chinese" },
+  { key: "Danish", text: "Danish", value: "Danish" },
+  { key: "Dutch", text: "Dutch", value: "Dutch" },
+  { key: "English", text: "English", value: "English" },
+  { key: "French", text: "French", value: "French" },
+  { key: "German", text: "German", value: "German" },
+  { key: "Greek", text: "Greek", value: "Greek" },
+  { key: "Hungarian", text: "Hungarian", value: "Hungarian" },
+  { key: "Italian", text: "Italian", value: "Italian" },
+  { key: "Japanese", text: "Japanese", value: "Japanese" },
+  { key: "Korean", text: "Korean", value: "Korean" },
+  { key: "Lithuanian", text: "Lithuanian", value: "Lithuanian" },
+  { key: "Persian", text: "Persian", value: "Persian" },
+  { key: "Polish", text: "Polish", value: "Polish" },
+  { key: "Portuguese", text: "Portuguese", value: "Portuguese" },
+  { key: "Russian", text: "Russian", value: "Russian" },
+  { key: "Spanish", text: "Spanish", value: "Spanish" },
+  { key: "Swedish", text: "Swedish", value: "Swedish" },
+  { key: "Turkish", text: "Turkish", value: "Turkish" },
+  { key: "Vietnamese", text: "Vietnamese", value: "Vietnamese" }
+];
+
+export class BecomeInterpreterForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: "",
+      username: "",
+      password: "",
+      confirmationpassword: "",
+      languages: "",
+      userCreated: null
+    };
+  }
+
+  handleChange = event => {
+    this.setState({
+      [event.target.name]: event.target.value
+    });
+    //console.log(this.state);
+  };
+  handleSubmit = e => {
+    e.preventDefault();
+    const { password, confirmationpassword } = this.state;
+    // perform all neccassary validations
+    if (password !== confirmationpassword) {
+      alert("Passwords don't match");
+    } else {
+      // make API call
+      const interpreterRequest = {
+        method: "POST",
+        body: JSON.stringify({
+          email: this.state.email,
+          username: this.state.username,
+          password: this.state.password,
```

Comment:
You already have these two variable above, you can use them instead

Reply:
I used them